### PR TITLE
version updated

### PR DESCRIPTION
**Pull request by asulwer**
1. RulesEngine.cs - UpdateErrorMessage, wrapped block in using statement
2. removed unused using statements
3. updated RulesEngine version to include changes from issue [#75](https://github.com/asulwer/RulesEngine/issues/75)
4. **targeting NetStandard2.0 AND 2.1**

### CODE REVIEW

**Comment by asulwer** [comment]
fyi: if you upgrade to VS 2022 17.12 it 'removes' Net 8 and installs Net 9.  project would not build because of the removal of Net 8

**Comment by JasonBoggsAtHMSdotCom** [comment]
Out of curiosity, what prompted you to support netstandard2.0? I was going to propose this due to integration requirements with an older app I'm working with, but looks like you already took care of this.

File: src/RulesEngine/Models/ImplicitObject.cs [hunk 1]
```diff
@@ -4,16 +4,11 @@
...
 
-using System.Diagnostics.CodeAnalysis;
-
 namespace RulesEngine.Models
 {
     [ExcludeFromCodeCoverage]
     public class ImplicitObject
     {
-
-        #region Implicit Operators
```
Comment (JasonBoggsAtHMSdotCom) [inline src/RulesEngine/Models/ImplicitObject.cs:15]: Do we not want region blocks?

Comment (asulwer) [reply src/RulesEngine/Models/ImplicitObject.cs:15]: the region wraps one block of code.  in my opinion, it isn't necessary

**Comment by asulwer** [comment]
> Out of curiosity, what prompted you to support netstandard2.0? I was going to propose this due to integration requirements with an older app I'm working with, but looks like you already took care of this.

i saw that you had it enabled in your branch.  what can it hurt supporting it?

**Review by RenanCarlosPereira — APPROVED** [review]

**Comment by RenanCarlosPereira** [comment]
All good thanks

**Comment by JasonBoggsAtHMSdotCom** [comment]
> > Out of curiosity, what prompted you to support netstandard2.0? I was going to propose this due to integration requirements with an older app I'm working with, but looks like you already took care of this.
> 
> i saw that you had it enabled in your branch. what can it hurt supporting it?

Works for me.  thanks!

**Pull request merged**